### ISSUE TITLE
List of Pokemob Attacks that need implementing

### ISSUE THREAD

**Issue by Thutmose** [Pokecube-Development/Pokecube-Issues-and-Wiki#337]
# Master list of known un-implemented/bugged moves

-  [ ] placeholder

**Comment by deathxf** [comment]
Aura Wheel Morpeko's unique attack changes elements depending on the morpeko mode, electric to full belly mode or to hungry mode like Multi-Attack from Silvally

**Comment by BlackRainGER** [comment]
Steahltrocks  (basically all pokemon attack wichs tend to not directly impact, but during rotations of multiple pokemons  (i think Curse does work as the only attack of that kind) but that's a bigger work in progress projekt i guess, the entire " 2 v 2 abilitie set up has no real purpose, even 

stat decreasings like 
Leer, tailwhipe etc, only hold for 1 round and are unable to connect multiple pokemon of the same trainer in a row as an ongoing fight

**Comment by Liviy56** [comment]
Hi! I wanted to say the shiny commands for pokecube 1.16 do not work thanks!

**Comment by Thutmose** [comment]
> Hi! I wanted to say the shiny commands for pokecube 1.16 do not work thanks!

this isn't the correct issue for that, also, what "shiny commands"?

**Comment by crtoonmastr** [comment]
Bestow shoots a particle at the target and does no damage but does not transfer the held item

**Comment by UnLuCkYcHaRrnS** [comment]
Aqua Ring does nothing but has an animation

Meteor Assault, Glacial Lance, and Astral Barrage have no animation. Glacial Lance and Astral Barrage also have only 25 power, when they are meant to have 130 and 120 power, respectively.

Life Dew and Lock-On do nothing and have no animation. Lock-On may actually do something, but the duration does not seem to permit another move to be used reliably before Lock-On expires. After many attempts, I was not able to connect Guillotine or Zap Cannon at an increased rate immediately after using Lock-On. I do not see a moves entry for it, so it probably just does nothing.

Plasma Fists and Astral Barrage TMs have improper string titles

**Comment by UnLuCkYcHaRrnS** [comment]
V-create is a signature move of Victini that was only added to its learnset in gen 8, prior to that it was only obtainable by attending a Nintendo event to receive a gift Victini that knows the move. Consequently, V-create does not exist in Victini's learnset within Pokecube, while it exists as a TM item within Pokecube (I found one). Therefore, all instances of V-create TMs are unusable. This move should be added to Victini's learnset as an innate (level 1) move, as it is in gen 8. May also need a balance pass as despite being a non-Z-move melee attack that lower's the user's defense, special defense, and speed, it has 180 power and 95% accuracy, requires no recharge and causes no recoil damage.

I've also found a move tutor move, Meteor Beam, which was introduced in gen 8, and many pokemobs that should be able to learn it cannot because it likewise does not appear on their learnset list in the Pokewatch, so all gen 8 move tutor moves are only learnable by gen 8 pokemobs.

This is pointing to a larger integration problem with the gen 8 allotment into the overall Pokecube moves system, which appears to primarily run on gen 7 data.

**Comment by crtoonmastr** [comment]
Listing Heal Pulse here so they're all in one place
Described previously in #488 
TL;DR: Should heal target (not user) by 50% of its max HP

**Comment by DogeWithChanga** [comment]
Mist Ball. The signature move of Latias. I think it got put in wrong as mist (myabe), but is unconfirmed.
